study.md

```diff
@@ -44,21 +44,21 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+A view is a HTML fragment. The view state is the view currently being presented to the user on the page, while a router is a conduit which allows movement from one view to another.
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+A single page application is far simpler to create, requiring less Js code.
 ```
```

Comment:
They are not always necessarily easier to create, but many JavaScript frameworks do streamline this process. They usually perform better and are easier to maintain!